Order.Management/Order.cs

```diff
@@ -1,20 +1,17 @@
-﻿using System;
+﻿using Order.Management.ToyBlocks;
 using System.Collections.Generic;
-using System.Text;
 
 namespace Order.Management
 {
-    abstract class Order
+    public class Order
     {
         public string CustomerName { get; set; }
         public string Address { get; set; }
-        public string DueDate { get; set; }
+        public string DueDate { get; set; } // TODO: should probably be DateTime
         public int OrderNumber { get; set; }
-        public List<Shape> OrderedBlocks { get; set; }
+        public List<Shape> OrderedBlocks { get; set; } = new List<Shape>();
 
-        public abstract void GenerateReport();
-
-        public string ToString()
+        public override string ToString()
```

Comment:
Prefer not to override the .net class `ToString()` method. Rather just have a `Print()` method or something like that